### PR TITLE
refactor: refactor bucks function with new class based on oeFormatMoney

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #

### CODE REVIEW

File: interface/patient_file/encounter/superbill_custom_full.php [hunk 1]
```diff
@@ -41,18 +41,6 @@ function ffescape($field)
     return trim($field);
```
Comment (bradymiller) [inline interface/patient_file/encounter/superbill_custom_full.php:41]: does this need a `use OpenEMR\Common\Utils\FormatMoney;`

File: interface/patient_file/front_payment.php [hunk 2]
```diff
@@ -70,25 +70,6 @@
         <script src=<?php echo $script; ?> charset="utf-8"></script>
     <?php } ?>
```
Comment (bradymiller) [inline interface/patient_file/front_payment.php:71]: does this need a `use OpenEMR\Common\Utils\FormatMoney;`

File: interface/reports/appt_encounter_report.php [hunk 3]
```diff
@@ -68,13 +68,6 @@ function postError($msg)
     $errmsg .= text($msg);
 }
```
Comment (bradymiller) [inline interface/reports/appt_encounter_report.php:69]: does this need a `use OpenEMR\Common\Utils\FormatMoney;`

File: interface/reports/collections_report.php [hunk 4]
```diff
@@ -172,14 +172,6 @@
     $grand_total_agedbal[$c] = 0;
 }
```
Comment (bradymiller) [inline interface/reports/collections_report.php:173]: `use OpenEMR\Common\Utils\FormatMoney;`

Comment (bradymiller) [reply interface/reports/collections_report.php:173]: same for below scripts

File: src/Common/Utils/FormatMoney.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<?php
```
Comment (bradymiller) [inline src/Common/Utils/FormatMoney.php:1]: tenor99

**Comment by stephenwaite** [comment]
testing well

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
2DV(2)

**Pull request merged**